index.html

```diff
@@ -181,9 +187,9 @@
             const deleteButton = document.createElement("button");
             deleteButton.textContent = "Delete";
             deleteButton.classList.add("btn", "waves-effect", "waves-light", "delete-button");
-            deleteButton.innerHTML = '<i class="material-icons">clear</i>';
+            deleteButton.innerHTML = '<i class="material-icons">delete</i>';
             deleteButton.addEventListener("click", () => {
-                app.removeChild(card);
+                card.removeChild(card);
```

Comment:
In the deleteButton event listener, use cards.removeChild(card) instead of app.removeChild(card) to ensure the correct parent element is targeted: